Analyzers/src/ExampleRun.C

```diff
@@ -331,8 +331,9 @@ void ExampleRun::executeEventFromParameter(AnalyzerParameter param){
 
   int n_bjet_deepcsv_m=0;
   int n_bjet_deepcsv_m_noSF=0;
+
   for(unsigned int ij = 0 ; ij < jets.size(); ij++){
-    
+    cout << event << endl;
```

Comment:
:D

Reply:
Ah, i forgot to commit before pushing that change. Its now removed.